sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-bike-share/auto-ml-forecasting-bike-share.ipynb

```diff
@@ -294,7 +321,7 @@
     "\n",
     "**Target column** is what we want to forecast.\n",
     "\n",
-    "**Time column** is the time axis along which to predict."
+    "**Time column** is the time axis along which to predict (we have defined it when we have split the data)."
```

Comment:
nit: we defined it when we split the data

Reply:
Fixed.